packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -602,15 +614,26 @@ export class StatsAnalyzer extends EventsScope {
             `StatsAnalyzer:index#compareLastStatsResult --> No video RTP packets sent`,
             currentStats.totalPacketsSent
           );
-        } else {
+        } else if (this.lastEmittedStartStopEvent.video.local !== EVENTS.LOCAL_MEDIA_STOPPED) {
           if (
             currentStats.framesEncoded === previousStats.framesEncoded ||
             currentStats.framesEncoded === 0
           ) {
+            this.lastEmittedStartStopEvent.video.local = EVENTS.NO_VIDEO_ENCODED;
```

Comment:
I really don't like how we're now setting lastEmittedStartStopEvent to something else than MEDIA_STARTED or MEDIA_STOPPED. The name lastEmittedStartStopEvent clearly says that it should be only started/stopped event and also lastEmittedStartStopEvent  should only ever by updated inside emitStartStopEvents(), Otherwise it's easy to introduce bugs. The whole point of emitStartStopEvents and lastEmittedStartStopEvent is to make sure that we never send STOPPED without STARTED or never send STARTED or STOPPED twice in a row etc. With the changes in this PR, the code becomes more convoluted and I'm not sure if it's guaranteed to still work correctly.

Reply:
Removed it . This was just an intermediate state for lastEmittedStartStopEvent.video.local.